docs/development.md

```diff
@@ -38,6 +38,9 @@ following:
  ❯ examples } ⊢ [OK] [143s]
 ```
 
+Note: `./scripts/docker_pull` and `./scripts/docker_run` will need `sudo` if not
```

Comment:
As a side note, I think at the moment we mostly use sudoless Docker (at least I do), but this is probably not a good practice, and we should instead require sudo, since the Docker daemon runs as root anyways. Anyways we can talk about that separately, and update the scripts if necessary.

@mariaschett  @conradgrobler @ipetr0v @rbehjati any opinions either way?